sklearn/preprocessing/data.py

```diff
@@ -231,9 +235,12 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
         Per feature relative scaling of the data.
     """
 
-    def __init__(self, feature_range=(0, 1), copy=True):
+    def __init__(self, feature_range=(0, 1), copy=True,
+                 continuous_features=None):
         self.feature_range = feature_range
         self.copy = copy
+        self.continuous_features_ = continuous_features
```

Comment:
Thank you for the feedback, @glouppe. I put the underscore in `continuous_features_` because in the case that a user doesn't specify which columns would be scaled, this parameter would have to be learned in `fit`.

I'm basing this naming convention from a previous discussion I had here:
https://github.com/scikit-learn/scikit-learn/pull/4801#discussion-diff-31586438